routes/baseDBMethods.js

```diff
@@ -22,6 +24,35 @@ function handleDocResponse(ctx, doc) {
   }
 }
 
+function proccessDatesRangeParameters(ctx) {
+  const startDate = parseInt(ctx.query.startDate, 10) || null;
+  const endDate = parseInt(ctx.query.endDate, 10) || null;
+
+  const datesFilter = {};
```

Comment:
Does eslint allow using `let` here instead of `const`?
You really never reassign to this variable, but you change its content through its fields, so I don't feel comfortable with it being a `const`. But maybe it's a common JS convention and I'm the one who's wrong here...

Reply:
I understand you, I wrote it as 'let' but ESlint WANT IT TO BE CONST.
You made me traumatic about ESlint just so you know.